model2vec/model_card_template.md

```diff
@@ -28,20 +28,6 @@ model = StaticModel.from_pretrained("{{ model_name }}")
 embeddings = model.encode(["Example sentence"])
 ```
 
-Alternatively, you can distill your own model using the `distill` method:
-```python
-from model2vec.distill import distill
-
-# Choose a Sentence Transformer model
-model_name = "BAAI/bge-base-en-v1.5"
-
-# Distill the model
-m2v_model = distill(model_name=model_name, pca_dims=256)
-
-# Save the model
-m2v_model.save_pretrained("m2v_model")
-```
-
```

Comment:
I actually like this bit, it helps push people interested in the work to exploring it themselves. My comments regarding the "distillation should be secondary in the model card" was referring to:

> Model2Vec distills a Sentence Transformer into a small, static model.
> This model is ideal for applications requiring fast, lightweight embeddings.

This first sentence isn't really useful for someone just looking at the model card. Perhaps a better first sentence is:

> This Model2Vec model is a distilled version of a Sentence Transformer that uses static embeddings, allowing text embeddings to be computed orders of magnitude faster inference on both GPU and CPU. It is designed for applications where computational resources are limited or where real-time performance is critical.

Or you can even use the Jinja template:

> This Model2Vec model is a distilled version of {% if base_model %}the [{{ base_model }}](https://huggingface.co/{{ base_model }}){% else %}a{% endif %} Sentence Transformer that uses static embeddings, allowing text embeddings to be computed orders of magnitude faster inference on both GPU and CPU. It is designed for applications where computational resources are limited or where real-time performance is critical.

And then you'll get something like:
> This Model2Vec model is a distilled version of the [BAAI/bge-large-en-v1.5](https://huggingface.co/BAAI/bge-large-en-v1.5) Sentence Transformer that uses static embeddings, allowing text embeddings to be computed orders of magnitude faster inference on both GPU and CPU. It is designed for applications where computational resources are limited or where real-time performance is critical.

Reply:
This is a nice suggestion! I updated the model card (added back the distillation part, and implemented your suggestion).